screens/ChooseLocationScreen.tsx

```diff
@@ -25,7 +25,7 @@ import { findCurrentLocation, findPlace } from '../context/MapContext';
 import { Marker } from '../maps';
 import { getHotspotMarker } from '../models/Hotspot';
 import { getFormattedAddress, Location } from '../models/Location';
-import { EdgeInsets, Region, RootStackScreenProps } from '../types';
+import { EdgeInsets, RootStackScreenProps } from '../types';
```

Comment:
are these used anywhere?
Grayed out means that the import is not used anywhere.
Btw, we should implement some linter to check for such situations.

Reply:
`EdgeInsets` is used as a type for the `insets` parameter in the `getStyles` function. `RootStackScreenProps` is used as a type for the props received by this screen component.